src/erlcloud_s3.erl

```diff
@@ -1141,6 +1141,8 @@ aws_region_from_host(Host) ->
     case string:tokens(Host, ".") of
         ["s3", _, _] ->
```

Comment:
per http://docs.aws.amazon.com/general/latest/gr/rande.html s3.eu-central-1.amazonaws.com is a valid endpoint, as well.